docs/topics/set-operations.md

```diff
@@ -29,5 +28,18 @@ fun main() {
 ```
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
 
-Note that set operations are supported by `List` as well.
-However, the result of set operations on lists is still a `Set`, so all the duplicate elements are removed.
+These operations – `union`, `intersect`, and `subtract` – apply to `List` as well.
```

Comment:
WDYT about You can also apply all these set operations to `List`?
Or you can also apply union, intersect, and substarct operations to `List`?

Just want to make this sentence easier to read.